<launch>
    <node pkg="btraj" type="astar" name="astar" output="screen"/>
    <node pkg="btraj" type="tf_br" name="tf_broadcaster"/>
    <node pkg="btraj" type="b_traj" name="b_traj" output="screen">
        <param name="max_inflate_iter"       value="2000" />
        <param name="B_traj/order"           value="6" />
        <param name="B_traj/vx_max"          value="10.0" />
        <param name="B_traj/vx_min"          value="-10.0" />
        <param name="B_traj/vy_max"          value="10.0" />
        <param name="B_traj/vy_min"          value="-10.0" />
        <param name="B_traj/ax_max"          value="20.0" />
        <param name="B_traj/ax_min"          value="-20.0" />
        <param name="B_traj/ay_max"          value="20.0" />
        <param name="B_traj/ay_min"          value="-20.0" />
    </node>
    <node name="map_server" pkg="map_server" type="map_server" args="$(find btraj)/maps/map.yaml" />
    <node name="rviz" pkg="rviz" type="rviz" args="$(find btraj)/rviz/rviz.rviz" />
</launch>